CMakeLists.txt

```diff
@@ -31,6 +31,9 @@ else()
   message(FATAL_ERROR "Invalid build type '${CMAKE_BUILD_TYPE}'")
 endif()
 
+enable_testing()
```

Comment:
You can actually move that to `tests/CMakeLists.txt` since it's only needed there.

Reply:
IIRC this is not a good idea since then it is only possible to call ctest from `tests` (but not from the root directory).

Reply:
Ah right, forget it then :-)